web/vtadmin/src/components/routes/topology/Topology.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2021 The Vitess Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { orderBy } from 'lodash-es';
+import * as React from 'react';
+
+import { useClusters } from '../../../hooks/api';
+import { useDocumentTitle } from '../../../hooks/useDocumentTitle';
+import { DataTable } from '../../dataTable/DataTable';
+import { vtadmin as pb } from '../../../proto/vtadmin';
+import { DataCell } from '../../dataTable/DataCell';
+import { ContentContainer } from '../../layout/ContentContainer';
+import { WorkspaceHeader } from '../../layout/WorkspaceHeader';
+import { WorkspaceTitle } from '../../layout/WorkspaceTitle';
+import { Link } from 'react-router-dom';
+
+const TopologyLink: React.FC<{ clusterID: string }> = ({ clusterID, children }) => {
```

Comment:
It could be worth moving this to `components/links`, since I can definitely think of other places we'd want to link to a topology viewer

Otherwise, if you'd like to keep it in this file for now, $0.02 of bike shedding but I'd find this more readable simply inline:

```
                <DataCell>
                    <Link className="font-bold" to={`/topology/${cluster.id}`}>
                        View Topology
                    </Link>
                </DataCell>
```